sdk/options.go

```diff
@@ -22,18 +22,19 @@ type config struct {
 	tokenExchange           *oauth.TokenExchangeInfo
 	tokenEndpoint           string
 	scopes                  []string
-	policyConn              *grpc.ClientConn
-	authorizationConn       *grpc.ClientConn
-	entityresolutionConn    *grpc.ClientConn
+	policyConn              *grpc.ClientConn // Deprecated
```

Comment:
Question: does the deprecation notice work if you in-line it? I couldn't find a clear answer in the docs and it looks like all examples on the first two pages of code searches in GH are the line above: https://github.com/search?q=%22%2F%2F+deprecated%22+path%3A*.go&type=code&p=1

Reply:
I am honestly not sure. It was more for an internal reminder. I am really not sure if we should deprecate them or just rip them out.

Reply:
At this point, I think we need to defecate since we know there's external use.

Reply:
I just removed this internal fields and now just set them to coreConn